modules/contrib/site_verify/site_verify.admin.inc

```diff
@@ -180,11 +185,49 @@ function site_verify_edit_form($form, &$form_state, $record = array(), $engine =
 }
 
 /**
- * Validation callback; save the uploaded file and check file name uniqueness.
+ * Validation callback; sanitize metatag and save the uploaded file
+ */
+function site_verify_validate($form, &$form_state) {
+  if (isset($form_state['values']['meta']) && $form_state['values']['meta']) {
+    site_verify_sanitize_metatag($form, &$form_state);
+  }
+
+  if (isset($form_state['values']['file']) && $form_state['values']['file']) {
+    site_verify_validate_file($form, &$form_state);
+  } else {
+    $form_state['values']['file_contents'] = NULL;
+  }
+
+}
+
+/**
+ * Sanitize metatag input
+ */
+function site_verify_sanitize_metatag($form, &$form_state) {
+  $valid_metatag_set = FALSE;
+
+  $html = $form_state['values']['meta'];
+  $dom = new DOMDocument;
+  $dom->loadHTML($html);
+  //@TODO: If we only care about the first metatag... why foreach?  why not [0]?
+  foreach ($dom->getElementsByTagName('meta') as $tag) {
+    if ($tag->getAttribute('name') && $tag->getAttribute('content')) {
```

Comment:
I'm not sure what to do about https://github.com/CuBoulder/express/pull/1032/files#diff-80889f50f1ce356f247deb917ba29d29R198 Currently I'm only keeping the last metatag.  It would be easy enough to change this so it only allowed the first, but I'm not sure if we should update the field description to indicate that only a single metatag is allowed?